### ISSUE TITLE
Error in kl_loss function

### ISSUE THREAD

**Issue by suvaansh** [graviraja/pytorch-sample-codes#2]
For the kl_loss, it should be "0.5 * torch.sum(-torch.log(z_var) + z_mu**2 - 1.0 - z_var)" instead of kl_loss = 0.5 * torch.sum(torch.exp(z_var) + z_mu**2 - 1.0 - z_var).

You can refer here for maths behind it: https://stats.stackexchange.com/questions/7440/kl-divergence-between-two-univariate-gaussians

**Comment by suvaansh** [comment]
In the VAE code

**Comment by graviraja** [comment]
You can see the equation from the paper and from the official pytorch implementation [here](https://github.com/pytorch/examples/blob/master/vae/main.py). I just rearranged the terms in loss function.
